sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -93,6 +93,8 @@ def test_output_shape(Estimator, method, data, grid_resolution, features, kind):
     # - multi-task regressors
 
     est = Estimator()
+    if hasattr(est, "n_estimators"):
+        est.set_params(n_estimators=5)  # speed-up computations
```

Comment:
Given that this check is only checking shapes, I think we can push it to two estimators:

```suggestion
        est.set_params(n_estimators=2)  # speed-up computations
```